integration-tests/src/bin/clock_gettime.rs

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use libc::clock_gettime;
+
+fn main() {
+    let mut ts = libc::timespec {
+        tv_nsec: Default::default(),
+        tv_sec: Default::default(),
+    };
+
+    unsafe {
+        clock_gettime(libc::CLOCK_MONOTONIC, &mut ts as *mut _);
```

Comment:
Is there any kind of assertion we can make here to confirm our assumptions about this test and determine pass/fail?

Reply:
@connorkuehl Not yet. See #747.

Reply:
What we have currently is that this payload executed the syscall correctly and didn't crash and didn't hang. #747 will confirm that the actual outputs are correct.

Reply:
Makes sense. Thanks!